src/App.js

```diff
@@ -1,19 +1,51 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
```

Comment:
Small nit, since L1 and L5 both import classes/functions from react, you can combine them in the same line:
```
import React, { useState } from 'react';
```